src/framebuffer/core.rs

```diff
@@ -75,10 +97,16 @@ impl<'a> framebuffer::FramebufferBase<'a> for Framebuffer<'a> {
             default_font,
             var_screen_info,
             fix_screen_info,
+            swtfb_client,
         }
     }
 
     fn set_epdc_access(&mut self, state: bool) {
+        if self.swtfb_client.is_some() {
+            // Not catched in rm2fb => noop
```

Comment:
```suggestion
            // Not handled in rm2fb => noop
```

Reply:
I believe that "catched" describes it better here. Since the ioctl wrapper didn't return its own result, the ioctl is passed on to whereever [here](https://github.com/ddvk/remarkable2-framebuffer/blob/1e288aa9cae24f7dda469bfaa32d92def90ba0e9/src/client/main.cpp#L232).

So it's not only just not handles, but the ioctl could technically still go through to the device/file if it were to be supported. Maybe stating something like "catched/handled" instead?

Reply:
Well it's just that past tense of "catch" is "caught". Do whatever :)

Reply:
Should be okay now with commit 79816f01.